pkg/project/project.go

```diff
@@ -110,9 +110,6 @@ func (p *Project) CommonSetupSteps() error {
 func (p *Project) CommonTearDownSteps() error {
 	p.logger.Log("info", "starting common teardown steps")
 	steps := []Step{
-		Step{
-			Run: "helm delete --purge sonobuoy-chart || true",
-		},
```

Comment:
For my interest why don't we need this anymore?

Reply:
Sorry, this change is not related to this PR at all, we removed sonobuoy support a long time ago from e2e-harness, we are getting `Error: release: "sonobuoy-chart" not found` on cleanup, like here https://circleci.com/gh/giantswarm/aws-operator/4498

Reply:
Cool thanks for the explanation. Good to remove this if it's not used.